### DISCUSSION TITLE
Produce different voices for the same sentence

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#75] [Archived]
<USER>>>> ritesh.nitjsr01<br/> <DATE> [August 15, 2019, 7:34pm]<br/> <br/> Hi, *slash* <br/> I wanted to know if there is a way to generate different voice samples<br/> for the same sentence, using any of the pre-trained models ?<br/> <br/> Thanks<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/produce-different-voices-for-the-same-sentence]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [August 16, 2019, 9:51am]<br/> <br/> different speakers you mean ? If so, no there is not.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> nmstoker<br/> <DATE> [August 22, 2019, 7:06am]<br/> <br/> Out of interest, for the **same** speaker, would the same text always<br/> produce the exact same audio output?<br/> <br/> Is there a way to produce slight variations, in the same way that people<br/> don't say a phrase in precisely the same way each time?<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [August 22, 2019, 1:15pm]<br/> <br/> you can enable the prenet dropout at inference<br/> <br/> [Archived Post]<br/>
